python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -98,13 +98,21 @@ def __init__(
         run_resources: Optional[dict[str, Any]] = None,
         run_ecs_tags: Optional[list[dict[str, Optional[str]]]] = None,
         propagate_tags: Optional[dict[str, Any]] = None,
+        task_definition_prefix: str = "run",
     ):
         self._inst_data = inst_data
         self.ecs = boto3.client("ecs")
         self.ec2 = boto3.resource("ec2")
         self.secrets_manager = boto3.client("secretsmanager")
         self.logs = boto3.client("logs")
 
+        self._task_definition_prefix = task_definition_prefix
+
+        check.invariant(
+            len(self._task_definition_prefix) <= 16,
```

Comment:
Can we double check somewhere that we're not bumping up against the max family name size?

Reply:
yeah, this method is pretty careful about that https://github.com/dagster-io/dagster/blob/4b711f5ee3c1b1088b911b67b136fc33baa92f08/python_modules/libraries/dagster-aws/dagster_aws/ecs/utils.py#L57-L75